src/applications/post-911-gib-status/containers/IntroPageSummary.jsx

```diff
@@ -0,0 +1,123 @@
+          </ul>
+          <p><strong>Note:</strong> If you use <strong>DS Logon</strong>, you’ll need to verify your identity online as part of our sign-in process.</p>
+          <p>Please see the blue sign-in module above to learn more about signing in, creating or upgrading an account, and verifying your identity.</p>
+        </div>
+      </div>
+      <h2 itemProp="name">What benefits information will I be able to see?</h2>
+      <div itemProp="acceptedAnswer" itemScope itemType="http://schema.org/Answer">
+        <div itemProp="text">
+          <p><strong>In your Post-9/11 GI Bill Statement of Benefits, you’ll see:</strong></p>
+          <ul>
+            <li>If you have any Post-9/11 GI Bill benefits</li>
+            <li>How much money you have left to use for your education or training</li>
+            <li>How much time you have left to use these benefits</li>
+          </ul>
+        </div>
+      </div>
+      <h2 itemProp="name">What if I’m having trouble seeing my Statement of Benefits?</h2>
+      <div itemProp="acceptedAnswer" itemScope itemType="http://schema.org/Answer">
+        <div itemProp="text">
+          <p>Your Post-9/11 GI Bill Statement of Benefits might not be available if one of these is true:</p>
+          <ul>
+            <li>The name on the account you’re signed in with doesn’t exactly match the name we have in our Post-9/11 GI Bill records.</li>
+            <li>We’re still processing your education benefits application, so we haven’t created a record yet for you. We usually process applications within 60 days. If you applied less than 60 days ago, please check back soon.</li>
+            <li>You haven’t applied yet for Post-9/11 GI Bill education benefits. <br/>
+              <a href="/education/apply/">Apply for education benefits</a>.</li>
+            <li>You’re not eligible for Post-9/11 GI Bill benefits.</li>
+            <li>You’re trying to access the tool during its scheduled downtime. This tool is available Sunday through Friday, 6:00 a.m. to 10:00 p.m. (ET), and Saturday 6:00 a.m. to 7:00 p.m. (ET).</li>
+          </ul>
+          <p>If none of the above situations applies to you, and you think your Statement of Benefits should be here, please call the VA.gov Help Desk at <a href="tel:+18555747286">1-855-574-7286</a>. We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET)</p>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function mapStateToProps(state) {
+  return {
+    isLoggedIn: userSelectors.isLoggedIn(state),
+    isLOA3: userSelectors.isLOA3(state),
+    isProfileLoading: userSelectors.isProfileLoading(state)
+  };
+}
+
+const mapDispatchToProps = {
+  toggleLoginModal: navActions.toggleLoginModal
+};
+
+const BrandConsolidationSummaryContainer = connect(mapStateToProps, mapDispatchToProps)(BrandConsolidationSummary);
+
+export default BrandConsolidationSummaryContainer;
```

Comment:
I went ahead and added the sign-in button functionality. This work was overdue after getting tied up in a bunch of edge cases, so I thought it would be worth overdelivering a little bit.